src/main/scala/nl/knaw/dans/api/sword2/package.scala

```diff
@@ -43,6 +43,8 @@ package object sword2 {
 
   case class BagStoreSettings(baseDir: String, baseUrl: String)
 
+  case class PropertiesResources(bagId: String, filePaths: List[String])
```

Comment:
Instead of bagId and filePaths it should be bagUri and fileUris. When sending an update, the client will include URIs in the `fetch.txt`.